docs/2.9.0/docs/tools/daml-shell/index.rst

```diff
@@ -3,8 +3,30 @@
 
 .. enterprise-only::
 
-Daml Shell
-##########
+.. _daml-shell-header:
+
+Daml Shell (``daml-shell``)
+###########################
+
+Daml shell is a terminal-based application that is used to inspect a Daml
+ledger by connecting to a live PQS datastore.  It can examine current or
+historical states of the ledger by querying ledger events, transactions, and
+contracts in a user-friendly manner.  The Daml Shell CLI  includes extensive
+help via the help command; each command having its own detailed help support.  
+
+Some of the actions that Daml Shell supports are:
+
+* Find a specific contract and display it. For example, if you have a contract-id, Daml Shell allows you to inspect the corresponding contract using the contract command.
+* Find all events related to a transaction ID.  A transaction is displayed as a series of ledger events (creates, archives, and exercises). 
+* Auto-completion for identifiers such as contract-ids, fully qualified names, and package IDs .
+* List active, inactive, or all contracts for a template using a template FQN as an argument.
+* Apply query and filter facilities to commands to manage the output.
+* Use the diff command to highlight the delta between two contract-ids for the same template.
```

Comment:
> The `diff` command isn't in the list in the Commands section -- should it be added?

Or do you mean the `compare-contracts` command rather than `diff`?

Reply:
I changed 'diff' to 'compare-contracts'.  I thought I did that before.  Thanks for catching that.